### PR TITLE
Added Reducers & Actions PR

### PR DESCRIPTION
**Pull request by WalterOkumu**
- Added Redux Toolkit
- Added Actions & Reducers

### CODE REVIEW

File: src/redux/categories/index.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
+// categories.js
+
+// Actions
+const CHECK_STATUS = 'CREATE';
```
Comment (brytebee) [inline src/redux/categories/index.js]: - [x] **Required:** You are required to use `duck's pattern` in defining the actions of the reducer.

File: src/redux/books/index.js [hunk 2]
```diff
@@ -0,0 +1,45 @@
+// books.js
+
+// Actions
+const CREATE = 'CREATE';
+const UPDATE = 'UPDATE';
+const DELETE = 'DELETE';
+const LOAD = 'LOAD';
```
Comment (brytebee) [inline src/redux/books/index.js]: - [x] **Required:** You are required to use `duck's pattern` in defining the actions of the reducer. [Read more.](https://github.com/erikras/ducks-modular-redux)

**Review by brytebee — CHANGES_REQUESTED** [review]
Hi, @WalterOkumu👋,

##

### Highlights💡
- Good commit message.✔️ 
- Good PR Summary.✔️
- Great UI design.✔️
- App functions as it should.✔️

##

Good job so far coming so far with the progress you've made! 💪💪
 
However, there are some issues that you still need to work on to go to the next project but you are almost there!



# Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Please, remember to tag me in your question so I can receive the notification.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @brytebee in your question so I can receive the notification.
### _**Use @brytebee on slack for faster responses.**_


**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by addod19 — APPROVED** [review]
### Hi @WalterOkumu, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

### Highlights
- [ ] Nice job implementing the ducks pattern :100: 


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**